doc/source/migration.rst

```diff
@@ -1,3 +1,60 @@
 =================
 Migration process
 =================
+
+  
+1. Cone docs-exports repository 
+
+  To start the Migration Process you have to clone the Docs-Exports
+  This can be achived with the following command 
+
+  git clone https://github.com/opentelekomcloud-docs/doc-exports
```

Comment:
you you wrap it (here and all other snippets) as code block in rst it will be rendered much better
```
.. code-block::

   git clone ...
```